src/App.css

```diff
@@ -1,4 +1,18 @@
 * {
-  size: 14px;
+  font-size: 14px;
```

Comment:
was there a reason you set the base font size for the website to 14px? I asked because `rem` is a multiplier of this value, and because this is set to 14px, the spacing for the hero banner doesn't work as expected (they should be 12px and 20px top and bottom, but because the base font size is 14, 0.75 and 1.25 results in 10.5 and 17.5px)
![image](https://github.com/user-attachments/assets/74469872-1fce-449d-8f26-785924a5992e)

You should change this to 16px, but if there's a reason you want to keep this value, it's best to do a list of selections (ie. li, p, a, etc.) instead of *